app/design/frontend/base/default/template/securesubmit/form.phtml

```diff
@@ -138,8 +138,11 @@ SecureSubmitMagento.init({
 <?php if ($_loggedIn && $allow_card_saving && count($customerStoredCards)): ?>
 <script type="text/javascript">
 var secureSubmitsavedcardselector = document.querySelector('.saved-card-selector input');
```

Comment:
@ttnnkkrr Coming back to this, `.saved-card-selector input` will potentially return multiple nodes depending on individual consumers. We [loop through all saved cards](https://github.com/hps/heartland-magento-extension/blob/master/app/design/frontend/base/default/template/securesubmit/form.phtml#L23) when creating these nodes, so there should be a 1-to-1 match.